src/pages/subjects/Subjects.tsx

```
@@ -99,7 +95,7 @@ const Subjects = () => {
       subjects.map((item: SubjectInterface) => {
         return (
           <CardWithLink
-            description={`${item.totalOffers[oppositeRole]} ${t(
+            description={`${item} ${t(
               'categoriesPage.offers'
             )}`}
```

ESLint

[error] Replace `⏎··············'categoriesPage.offers'⏎············` with `'categoriesPage.offers'`

(prettier/prettier)